src/pages/MetricDetail.svelte

```diff
@@ -196,7 +196,8 @@
           <td>
             GLAM
             <HelpHoverable
-              content={'View this metric in Glean Aggregated Metrics'} />
+              content={'View this metric in Glean Aggregated Metrics'}
```

Comment:
```suggestion
              content={'View this metric in the Glean Aggregated Metrics (GLAM) dashboard'}
```
I think it's a little bit clearer this way.